WalletWasabi.Fluent/ViewModels/Wallets/Home/History/Details/CoinJoinDetailsViewModel.cs

```diff
@@ -44,6 +42,10 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 			                .Connect()
 							.Subscribe(_ => Update())
 							.DisposeWith(disposables);
+
+		_wallet.Transactions.RequestedFeeArrived
+							.Subscribe(_ => Update())
+							.DisposeWith(disposables);
```

Comment:
`_wallet.Transactions.RequestedFeeArrived` should be merged with `_wallet.Transactions.Cache.Connect()` to avoid concurrency issues.

Reply:
I got rid of it entirely.